example/07_conv2d_fwd_bias_relu_add/conv2d_fwd_xdl_bias_relu_add.cpp

```diff
@@ -83,154 +86,164 @@ using ReferenceConvFwdInstance =
                                                                      WeiElementOp,
                                                                      OutElementOp>;
 
-int main(int argc, char* argv[])
+void PrintUseMsg()
```

Comment:
function name should use print_use_msg

Function name in Coding style guide
https://github.com/ROCmSoftwarePlatform/composable_kernel/wiki/Coding-Style#naming-style

Please do a string replacement on the function name